charts/woodpecker/README.md

```diff
@@ -0,0 +1,49 @@
+# Woodpecker Helm Charts
+
+This repository contains the helm charts of the [Woodpecker](https://woodpecker-ci.org) project.
+
+![Version: 0.4.2](https://img.shields.io/badge/Version-0.4.2-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.2](https://img.shields.io/badge/AppVersion-1.0.2-informational?style=flat-square)
+
+**Source Code**: <https://github.com/woodpecker-ci/woodpecker>
+
+## Installation
```

Comment:
Should be part of the top level readme I guess

Reply:
I kept the top-level README tidy and only added links to the chart README. That's also how others like bitnami do it, i.e. have the chart specific READMEs at the root of the chart. And in this case, the installation instructions would go here.

Reply:
Okay for now. Maybe we can add an extra link to the top-level to direct users to this chart as I assume users will read the top-level first and wont know there is more info on the woodpecker chart.

Reply:
Links to the chart READMEs are already present in the top-level README. Or do you mean something else and we misunderstand each other?

Reply:
I was mainly referring to new users and how they would know where to click to get started. That's why I would personally leave some more context in the top-level Readme, but we can still improve that later again in a new PR